amethyst_renderer/src/renderer.rs

```diff
@@ -330,3 +365,76 @@ fn init_backend(wb: WindowBuilder, el: &EventsLoop, config: &Config) -> Result<B
 
     Ok(Backend(dev, fac, main_target, win))
 }
+
+/// Creates the Direct3D 11 backend.
+#[cfg(all(feature = "d3d11", target_os = "windows"))]
+fn init_existing_backend(window: &Window) -> Result<(Device, Factory, Target)> {
```

Comment:
Is this correct way to recreate render target?
For example there is [a function for openGL](https://github.com/gfx-rs/gfx/blob/v0.16/src/window/glutin/src/lib.rs#L153) and [similar one for DX](https://github.com/gfx-rs/gfx/blob/v0.16/src/window/dxgi/src/lib.rs#L194)

Reply:
That looks great!  Thanks!

Reply:
Hmm so it appears that only works for targets with a DepthStencilBuffer.  Should we make a DepthStencilBuffer required?

Reply:
No, stencil should be when needed imho

Reply:
You use those fuctions only for main target. And right now it has both unconditionally